packages/tools/devtools/devtools-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -33,6 +40,10 @@ export interface AudienceHistoryTableProps {
 export function AudienceHistoryTable(props: AudienceHistoryTableProps): React.ReactElement {
 	const { audienceHistoryItems } = props;
 
+	const clientIdTooltipText =
```

Comment:
We should probably de-duplicate this between the two files, since it's the same text. It might be worth adding a module for common tooltip text or something.